### DISCUSSION TITLE
How to get badge

### DISCUSSION THREAD

**Discussion by takasa0220** [takasa0220/es6#7] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tahj-scott** [comment, marked as answer]
Easy way to get a brain badge...
